### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by lexbarba**
Precourse projects added

### CODE REVIEW

**Comment by danieltolmon** [comment]
The alphabet is very fun, but I would add the name and the ranking

File: staff/alex-barba/precourse/calculadora/style/style.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+button {
+	width: 100px;
+	font-size: 20px;
+	padding: 15px;
+	background: transparent;
+	color: white;
+}
+
+#doit {
+	height: 232px;
```
Comment (martimalek) [inline staff/alex-barba/precourse/calculadora/style/style.css]: Hey! Try with a height of 240px, maybe it looks more compact...

Great shortcut in main.js though!

**Comment by lexbarba** [comment]
> He encontrado estos errores a medida que he realizado algunos "shots"
> image
> 
> Brutal proyecto! el que más me ha gustado!

Gracias! En principio la consola ya está limpia.

File: staff/alex-barba/precourse/calculadora/style/style.css (hunk 1 above)
Comment (lexbarba) [reply staff/alex-barba/precourse/calculadora/style/style.css]: Thx!

**Comment by lexbarba** [comment]
> The alphabet is very fun, but I would add the name and the ranking

Thanks Daniel! Added username and ranking

**Pull request merged**